requirements/uitest.txt

```diff
@@ -0,0 +1,10 @@
+execnet==1.4.1
+fxapom==1.7.1
+py==1.4.31
+pyjwt==1.4.0
+PyPOM==1.0
+pytest==2.9.1
+pytest-instafail==0.3.0
+pytest-selenium==1.2.1
+pytest-variables==1.4
+pytest-xdist==1.14
```

Comment:
actually, for our travis tests we're using the `travis.txt` (which really is just `docker.txt` with a different index because of platform dependencies).

And maybe, now that I'm thinking about it (or... it's the chocolate I'm eating :chocolate_bar:)... the uitests are run against the container right? Then it really doesn't make much sense to have the whole stack of dependencies installed too. It might make it easier to run the tests on local development though which imho should be a primary (or at least secondary) goal too. tbh, whatever works best for you.